README.md

```diff
@@ -12,115 +12,131 @@ npx knex migrate:latest
-  | id | Integer | Unique identifier for each user |
-  | first_name | String | User first name |
-  | last_name | String | User last name |
-  | email | String | User email |
-  | username | String | User login name |
-  | image | String | User avatar image |
-  | created_at | Timestamp | Time account created |
-  | password_hash | String | Hash/Password of user to login |
-  | reset_expires_at | Timestamp | Time of reset expiry |
-  | reset_password_key | String | Password key for reset |
-
-  ## projects
 
-   | Column Name | Data Type | Purpose |
-   | --- | --- | --- |
-   | id | Integer | Unique identifier for each project |
-   | user_id | Integer | Unique identifier for each user |
-   | name | String | User's name |
-   | description | 
+### users
 
+| Column Name        | Data Type | Purpose                         |
+| ------------------ | --------- | ------------------------------- |
+| id                 | Integer   | Unique identifier for each user |
+| first_name         | String    | User first name                 |
+| last_name          | String    | User last name                  |
+| email              | String    | User email                      |
+| username           | String    | User login name                 |
+| image              | String    | User avatar image               |
+| created_at         | Timestamp | Time account created            |
+| password_hash      | String    | Hash/Password of user to login  |
+| reset_expires_at   | Timestamp | Time of reset expiry            |
+| reset_password_key | String    | Password key for reset          |
+
+## projects
+
+    // Please note '
+
+| Column Name | Data Type | Purpose                                |
+| ----------- | --------- | -------------------------------------- |
+| id          | Integer   | Unique identifier for einterchangeable |
+| user_id     | Integer   | Unique identifier for each user        |
+| name        | String    | User's name                            |
+| description | String    | Description of project                 |
+| topic       | String    | Topic of Event                         |
+| date_start  | String    | Start date of event                    |
+| date_end    | String    | End date of event                      |
+| image       | String    | Image for Event                        |
+
+## user projects
+
+| Column Name | Data Type | Purpose                                  |
+| ----------- | --------- | ---------------------------------------- |
+| user_id     | Integer   | Unique identifier for each user          |
+| project_id  | Integer   | Unique identifier for each project/event |
+| team_id     | Integer   | Unique identifier for each project/event |
+| user_id     | Integer   | FOREIGN user_id from user table          |
+| project_id  | Integer   | FOREIGN project_id from project table    |
 
- ## user projects
- | Column Name | Data Type | Purpose |
- | --- | --- | --- |
- 
+---
 
+## Testing
```

Comment:
❕ This would be a good place to add instructions for how to run tests for client and server side.